src/CarbonAware.CLI/test/unitTests/Commands/Emissions/EmissionsCommandTests.cs

```diff
@@ -25,7 +25,7 @@ public async Task Run_CallsAggregatorAndWritesResults()
             Rating = 100.7
         };
       
-        _mockEmissionsAggregator.Setup(agg => agg.GetEmissionsDataAsync(It.IsAny<CarbonAwareParameters>()))
+        _mockEmissionsHandler.Setup(handler => handler.GetEmissionsDataAsync(It.IsAny<string[]>(), null, null))
```

Comment:
Suggestion. Can you replace `null` keyword, with `It.IsAny<DateTimeOffset>()` instead?
It helps to clarify the data types been passed.

Reply:
done